packages/server/graphql/public/mutations/upgradeToTeamTier.ts

```diff
@@ -0,0 +1,93 @@
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import removeTeamsLimitObjects from '../../../billing/helpers/removeTeamsLimitObjects'
+import getRethink from '../../../database/rethinkDriver'
+import updateTeamByOrgId from '../../../postgres/queries/updateTeamByOrgId'
+import {analytics} from '../../../utils/analytics/analytics'
+import {getUserId} from '../../../utils/authorization'
+import publish from '../../../utils/publish'
+import setTierForOrgUsers from '../../../utils/setTierForOrgUsers'
+import setUserTierForOrgId from '../../../utils/setUserTierForOrgId'
+import standardError from '../../../utils/standardError'
+import hideConversionModal from '../../mutations/helpers/hideConversionModal'
+import {MutationResolvers} from '../resolverTypes'
+
+const upgradeToTeamTier: MutationResolvers['upgradeToTeamTier'] = async (
+  _source,
+  {orgId},
+  {authToken, dataLoader, socketId: mutatorId}
+) => {
+  const r = await getRethink()
+  const operationId = dataLoader.share()
+  const subOptions = {mutatorId, operationId}
+  const now = new Date()
+
+  // AUTH
+  const viewerId = getUserId(authToken)
+  const organization = await dataLoader.get('organizations').load(orgId)
+  const {stripeId, tier, activeDomain, name: orgName} = organization
+
+  if (!stripeId) {
+    return standardError(new Error('Organization does not have a stripe id'), {
+      userId: viewerId
+    })
+  }
```

Comment:
-1 We should check that the `stripeId` is attached to a valid subscription. Having a random `stripeId` and then calling `upgradeToTeamTier` shouldn't be enough to remove all restrictions and updating the tier.